src/scss/_main.scss

```diff
@@ -982,6 +983,11 @@ nav {
   .-inner {
     fill: var(--color-primary);
   }
+
+  &.-skip {
+    //here
```

Comment:
same with this?
```suggestion
```

Reply:
unless i'm mistaken i still see this in the latest changeset.